src/mappings/misconfigurations.json

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
I wonder if we should just install the `cloud_security_posture` integration and then we always get the latest mapping? we already have an install package util https://github.com/elastic/security-documents-generator/blob/387d5c681f60bdf4300d1e4294d248d69ab75df5/src/utils/kibana_api.ts#L241